var/spack/repos/builtin/packages/py-planetary-computer/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
Unless I missed something, shouldn't this end `2023` now?

Reply:
I stopped updating Spack since Spack stopped running my pipelines for 24 hrs after the PR was submitted if I update first. Is this something that needs to be changed manually? If the tests don't fail, does it matter what the copyright year says?